include/linux/soundwire/sdw_registers.h

```diff
@@ -114,12 +120,111 @@
 #define SDW_SCP_DEVID_4				0x54
 #define SDW_SCP_DEVID_5				0x55
 
+/* Both INT and STATUS register are same */
+#define SDW_SCP_SDCA_INT1			0x58
+#define SDW_SCP_SDCA_INT_SDCA_0			BIT(0)
+#define SDW_SCP_SDCA_INT_SDCA_1			BIT(1)
+#define SDW_SCP_SDCA_INT_SDCA_2			BIT(2)
+#define SDW_SCP_SDCA_INT_SDCA_3			BIT(3)
+#define SDW_SCP_SDCA_INT_SDCA_4			BIT(4)
+#define SDW_SCP_SDCA_INT_SDCA_5			BIT(5)
+#define SDW_SCP_SDCA_INT_SDCA_6			BIT(6)
+#define SDW_SCP_SDCA_INT_SDCA_7			BIT(7)
```

Comment:
The indentation is weird here.  Below SDCA_10.. onwards alignment is different than here.

Reply:
diff mirage, looks good when you view the file directly

Reply:
@plbossart I thought so too as well, but I checked out locally and this is how it looks in emacs:
```
#define SDW_SCP_SDCA_INT2»      »       »       0x59
#define SDW_SCP_SDCA_INT_SDCA_8»        »       »       BIT(0)
#define SDW_SCP_SDCA_INT_SDCA_9»        »       »       BIT(1)
#define SDW_SCP_SDCA_INT_SDCA_10»       »       BIT(2)
#define SDW_SCP_SDCA_INT_SDCA_11»       »       BIT(3)

```

So it does seem off.

Reply:
@kv2019i looks fine for me with emacs and vim.

Reply:
@plbossart Heh, sorry for the noise -- one cannot really argue if you checked both with emacs _and_ vim. :) It turns out you are right, indentation is correct. It turned out changes I made to whitespace package params in my .emacs messed up tab visualization in case tab is on character away from next tab stop (like INT_SDCA_8 above), so I see an extra tab worth of indentation.